requre/online_replacing.py

```diff
@@ -81,3 +84,38 @@ def _internal(*args, **kwargs):
         return _internal
 
     return decorator_cover
+
+
+def store(what: str, in_module: str):
+    """
+    Decorator which can be used to store calls of the function and
+    and replay responses on the next run.
+    """
+
+    def _store_inner(func):
+        return replace(
+            what=what, in_module=in_module, decorate=Simple.decorator_all_keys
+        )(func)
+
+    return _store_inner
+
+
+def store_requests(_func=None):
+    """
+    Decorator which can be used to store all requests to a file
+    and replay responses on the next run.
+
+    Can be used with or without parenthesis.
+    """
+
+    def decorator_cover(func):
+        return replace(
+            what="Session.send",
+            in_module="^requests$",
+            decorate=RequestResponseHandling.decorator(item_list=[]),
```

Comment:
this `decorator` with `item_list=[]` has same meaning as `decorator_plain`. But I'll improve it, add ther some filterinig to URL and these requests then will use URL as an key, but I have  to filter authentication from URL

Reply:
OK, but I would like to use this one so we can improve this shortcut in one place.

I'll add the following TODO:
- [ ] argument for filtering headers (If you implement it, I'll just forward the option.)